dev/devicelab/lib/framework/runner.dart

```diff
@@ -124,47 +122,3 @@ Future<VMIsolateRef> _connectToRunnerIsolate(Uri vmServiceUri) async {
     }
   }
 }
-
-Future<void> cleanupSystem() async {
```

Comment:
We should fix the gradle shutdown by disabling the gradle daemon on bots once they're on LUCI. Once we have a build server that is extra obsolete.